### ISSUE TITLE
Problems with Cypress 

### ISSUE THREAD

**Issue by Dassderdie** [hpi-sam/digital-fuesim-manv#525]
This issue tracks all our current problems with cypress (our end-to-end test framework).

Please create a separate issue if you want to discuss/work on any of the following.

* Cypress does not work with Node 18 & ES Modules #503
    * > When using cypress in a Node 18 project that uses ES Modules, cypress open is stuck in the loading animation while cypress run doesn't do anything. This is reproducible in a minimal setup (npm init, npm i typescript, npm i cypress).
* Add more e2e tests #207 (we currently only have one that tests the landing page)
* Visual Regression tests have problems with the used font #176 
    * See https://github.com/hpi-sam/digital-fuesim-manv/pull/174#issuecomment-1032599585 ff.
    * This is why we currently don't have any visual regression tests.

**Comment by Greenscreen23** [comment]
Cypress now works with node 18, we are adding more e2e tests in the linked pr and removing the visual regression tests as we didn't like their flakyness. I therefore think this issue can be closed once that pr is merged.
